x/superfluid/spec/12_minting.md

```diff
@@ -0,0 +1,37 @@
+<!--
+order: 12
+-->
+
+# Minting
+
+
+Superfluid module has the ability to arbitrarily mint and burn Osmo through the `bank` module. This is potentially dangerous so we strictly constrain it's ability to do so.
+This authority is mediated through the `mintOsmoTokensAndDelegate` and `forceUndelegateAndBurnOsmoTokens` keeper methods, which are in turn called by message handlers (`SuperfluidDelegate` and `SuperfluidUndelegate`) as well as by hooks on Epoch (`RefreshIntermediaryDelegationAmounts`) and Lockup (`SuperfluidDelegateMore`)
+
+## Invariant
+Each of these mechanisms maintains a local invariant between the amount of Osmo minted and delegated by the `IntermediaryAccount`, and the quantity of the underlying asset held by locks associated to the account, modified by `OsmoEquivalentMultiplier` and `RiskAdjustment` for the underlying asset. Namely that total minted/delegated = `GetTotalSyntheticAssetsLocked` * `GetOsmoEquivalentMultiplier` * `GetRiskAdjustment`
```

Comment:
Perhaps lets explain the `TotalSyntheticAssetsLocked` term? (Or make a sub-heading at the end, labeled `TotalSyntheticAssetsLocked`, and then say TODO.

Basically whenever a lock is superfluid staked, we increment a synthetic denom for underlying denom, validator pair. Then TotalSyntheticAssetsLocked sums over all locked synthetic denoms denom validator pairs (accounting for multipliers per denom correctly).